objectscale-manager/templates/objectscale-manager-app-configmap.yaml

```
@@ -48,24 +48,65 @@ data:
           timelimit: "5m"
           schedule: {{ default "0 * * * *" .Values.healthChecks.inventory.schedule | quote}}
           disabled: {{ default false .Values.healthChecks.inventory.disabled}}
+          args:
+            - -platform
+            - {{ .Values.global.platform}}
+            - -watch_ns
+{{- if eq .Values.global.watchAllNamespaces true}}
+            - "" 
+{{- else }}
+            - metadata.namespace
+{{- end }}
```

can we make this simpler.  if the argument is not there then watch all namespaces.
```suggestion
{{- if neq .Values.global.watchAllNamespaces true }}
            - -watch_ns
            - metadata.namespace
{{- end}}
```

---

just to keep this consistent with other watch namespace. We will provide the args with either empty string or it's own namespace.